common/beerocks/bwl/dwpal/base_wlan_hal_dwpal.cpp

```diff
@@ -825,6 +825,56 @@ bool base_wlan_hal_dwpal::refresh_radio_info()
     return true;
 }
 
+bool base_wlan_hal_dwpal::get_vap_type(const std::string &ifname, bool &fronthaul, bool &backhaul)
+{
+    char *reply              = nullptr;
+    size_t numOfValidArgs[1] = {0}, replyLen = 0;
+    char mode[16]                 = {0};
+    FieldsToParse fieldsToParse[] = {
+        {(void *)mode, &numOfValidArgs[0], DWPAL_STR_PARAM, "mesh_mode=", sizeof(mode)},
+        /* Must be at the end */
+        {NULL, NULL, DWPAL_NUM_OF_PARSING_TYPES, NULL, 0}};
+    std::string cmd = "GET_MESH_MODE " + ifname;
+
+    fronthaul = false;
+    backhaul  = false;
```

Comment:
I think there's no need to initialize these variables to `false`, since if this method returns `false` the values should be ignored anyway (nitpick)